src/monitoring/aws/rds/metrics.ts

```diff
@@ -0,0 +1,100 @@
+import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
+import * as cdk from '@aws-cdk/core';
+
+/**
+ * Metrics for RDS Aurora.
+ */
+export class RdsAuroraMetricFactory {
+  metricDmlThroughput(clusterIdentifier: string) {
+    const dbInsertThroughputMetric = new cloudwatch.Metric({
+      metricName: 'InsertThroughput',
+      namespace: 'AWS/RDS',
```

Comment:
Let's extract the namespace out in a constant.

Reply:
Will do!